exercise-book/src/nrf52-tools.md

```diff
@@ -175,8 +175,16 @@ Bus 002 Device 015: ID 1366:1051 <- J-Link on the nRF52840 Development Kit
 (...) random other USB devices will be listed
 ```
 
-✅ In the terminal run the following command from the [`nrf52-code/radio-app`](../../nrf52-code/radio-app) folder. This will build and run a simple program on the DK to test the set-up.
+✅ In the terminal run `cargo run --bin hello -- --allow-erase-all` from the [`nrf52-code/radio-app`](../../nrf52-code/radio-app) folder, to build and run a simple program on the DK to test the set-up.
```

Comment:
```suggestion
✅ In the terminal run `cargo run --bin hello` from the [`nrf52-code/radio-app`](../../nrf52-code/radio-app) folder, to build and run a simple program on the DK to test the set-up.
```

I made `--allow-erase-all` the default in https://github.com/ferrous-systems/rust-exercises/pull/133/commits/e18284d60b5da90346c91a1e9f6de0b270da7cd4